api/app/routers/hfi_calc.py

```diff
@@ -0,0 +1,86 @@
+""" Routers for HFI Calculator """
+import logging
+from typing import List
+from fastapi import APIRouter, Response, Depends
+import app
+from app import wildfire_one
+from app.auth import authentication_required, audit
+from app.schemas.hfi_calc import HFIWeatherStationsResponse, WeatherStationProperties,\
+    FuelType, FireCentre, PlanningArea, WeatherStation
+from app.db.crud.hfi_calc import get_fire_weather_stations
+
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(
+    prefix="/hfi-calc",
+    dependencies=[Depends(authentication_required), Depends(audit)]
+)
+
+
+@router.get('/fire-centres', response_model=HFIWeatherStationsResponse)
+# pylint: disable=too-many-locals
```

Comment:
Please either remove # pylint: disable=too-many-locals, or move it to line 23; If it's on module level scope, as you have it right now, it will end up enforcing the disable rule on a global scope.